src/core/Akka.Tests/Actor/ActorTelemetrySpecs.cs

```diff
@@ -296,7 +295,7 @@ public async Task ActorTelemetry_must_be_accurate_for_pool_router()
                 var telemetry = await subscriber.Ask<TelemetrySubscriber.GetTelemetry>(TelemetrySubscriber.GetTelemetryRequest.Instance);
                 // assert that actor start count is still 10
                 Assert.Equal(12, telemetry.ActorCreated);
-                Assert.Equal(110, telemetry.ActorRestarted);
+                Assert.Equal(10, telemetry.ActorRestarted);
```

Comment:
We will also need to backport this to v1.4 as well

Reply:
I wonder why we didn't caught this before... I guess we missed it amongst the racy unit test noise?

Reply:
added the unit test fix in #6376